Added CONTIBUTING.md (#26)
in-toto/in-toto.io: CONTRIBUTING.md

```sh
git clone https://github.com/in-toto/in-toto.io.git
```

### 2. Install Development Dependencies

Navigate to the project root directory and install the necessary dependencies with Yarn. If you do not have Yarn installed, please install it by following the instructions on the [Yarn installation guide](https://classic.yarnpkg.com/en/docs/install):

```sh
cd in-toto.io
yarn install
```

### 3. Install Hugo

We use Hugo to build the site. Follow the [Hugo installation guide](https://gohugo.io/installation/) to install Hugo on your system. 

### 4. Serve the Project

Once Hugo is installed, you can serve the project locally to see your changes in real-time.
The root of the directory contains a `Makefile` for automating development processes. The `make` CLI tool is usually installed by default on most systems (excluding Windows), but you can check if it is installed by running `make --version` on your terminal. If this command is unsuccessful, you will need to find the standard method for installing it for your system. For installing `make` on Windows, please see [here](https://gnuwin32.sourceforge.net/packages/make.htm).
Run the following command:

```sh
make serve
```

> This will start a local server at `http://localhost:XXXX/` where you can view the site.

Reviewer: There is a chance that people wonder why the server is not actually listing on port "XXXX". What about?

```suggestion
This will start a local server. You can view your site at the URL displayed in your terminal.
```

# Contributing to in-toto.io

Thank you for considering contributing to in-toto.io! Here are the steps to set up the project on your local machine.

## Quick Start Guide

To get started with contributing to in-toto.io, follow these steps:

### 1. Clone the Repository

Clone the in-toto.io git repository to your local machine:

```sh
git clone https://github.com/in-toto/in-toto.io.git
```

> ### 2. Install Development Dependencies

Reviewer 2: There is inconsistency in headers spacing. Running a markdown formatter on it might be nice.

Navigate to the project root directory and install the necessary dependencies with Yarn:

```sh
cd in-toto.io

> yarn install

Reviewer 2: You might need to add instructions for yarn install too.

### 4. Serve the Project
Once Hugo is installed, you can serve the project locally to see your changes in real-time. Run the following command:

```sh
make serve
```
This will start a local server at http://localhost:XXXX/ where you can view the site.

### 5. Review Contribution Guidelines
Before contributing, please review our [contribution guidelines](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md) to ensure that your code follows our style guidelines and is properly tested.

### 6. Sign the Developer Certificate of Origin (DCO)
All contributors must sign the Developer Certificate of Origin (DCO) by adding a "Signed-off-by" line to their commit messages. This indicates your acceptance of the DCO. You can do this by appending the following line to each commit message (see [git commit –signoff](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff)):

```sh
git commit -s -m "Your descriptive commit message"
```

### Need Help?
If you encounter any issues or have any questions, feel free to open an issue on the repository or reach out to the maintainers.

> #### Happy contributing!

Reviewer 2: Don't miss the standard empty line at the end.

# Contributing to in-toto.io

Thank you for considering contributing to in-toto.io! Here are the steps to set up the project on your local machine.

## Quick Start Guide

To get started with contributing to in-toto.io, follow these steps:

### 1. Clone the Repository

Clone the in-toto.io git repository to your local machine:

```sh
git clone https://github.com/in-toto/in-toto.io.git
```

### 2. Install Development Dependencies

Navigate to the project root directory and install the necessary dependencies with Yarn. If you do not have Yarn installed, please install it by following the instructions on the [Yarn installation guide](https://classic.yarnpkg.com/en/docs/install):

```sh
cd in-toto.io
yarn install
```

### 3. Install Hugo

We use Hugo to build the site. Follow the [Hugo installation guide](https://gohugo.io/installation/) to install Hugo on your system. Below are simplified steps for different operating systems:

> On macOS:

Reviewer 3: I'm leery of OS specific recommendations. For eg, the linux recommendation is tailored to ubuntu/debian/derivatives. I think pointing to the hugo website is sufficient.

```sh
brew install hugo
```

On Windows:

Download the latest release from the [Hugo Releases](https://github.com/gohugoio/hugo/releases) page, then extract it and add the executable to your PATH.

On Linux:

```sh
sudo apt-get install hugo
```

For more detailed instructions, please refer to the [Hugo documentation](https://gohugo.io/installation/).

### 4. Serve the Project

Once Hugo is installed, you can serve the project locally to see your changes in real-time. Run the following command:

```sh

> make serve

Reviewer 3: Can we assume `make` is installed? :smile:

Author: After installing hugo make command is itself running.

Reviewer 4: Here's an [example](https://github.com/in-toto/witness/blob/main/CONTRIBUTING.md?plain=1#L23-L38) that would pretty close to copy/paste.

```

> This will start a local server at http&#x3A; //localhost: XXXX/ where you can view the site.

Reviewer 2: If you change this url inside a code block you dont need to use this separation and looks clearer.

> ### 5. Review Contribution Guidelines

Reviewer 3: I'd put this and DCO up front as some things to keep in mind with contributing to this repo. Possibly, this should just point out to https://github.com/in-toto/community/blob/main/CONTRIBUTING.md.

Before contributing, please review our [contribution guidelines](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md) to ensure that your code follows our style guidelines and is properly tested.

### 6. Sign the Developer Certificate of Origin (DCO)

All contributors must sign the Developer Certificate of Origin (DCO) by adding a "Signed-off-by" line to their commit messages. This indicates your acceptance of the DCO. You can do this by appending the following line to each commit message (see [git commit --signoff](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff)):

```sh
git commit -s -m "Your descriptive commit message"
```

> ### Need Help?

Reviewer 3: ```suggestion
## Need Help?
```

If you encounter any issues or have any questions, feel free to open an issue on the repository or reach out to the maintainers.

> #### Happy contributing!

Reviewer 3: ```suggestion
**Happy contributing!**
```
